src/sql/workbench/contrib/notebook/browser/notebookExplorer/notebookExplorerViewlet.ts

```diff
@@ -314,6 +320,10 @@ export class NotebookExplorerViewPaneContainer extends ViewPaneContainer {
 				this.searchWidget.focus(false, true); // focus back to input field
 			}
 		}, onQueryValidationError);
+		let end = new Date().getTime();
+		this._telemetryService?.createActionEvent(TelemetryKeys.TelemetryView.Notebook, TelemetryKeys.NbTelemetryAction.SearchComplete)
```

Comment:
You don't want to capture any additional information such as # of results returned, how many different files those results were in, etc?